### ISSUE TITLE
Can't start indexer from already synced node 

### ISSUE THREAD

**Issue by RuslanGlaznyov** [anoma/namada-indexer#184]
If I have already synced the node and run the indexer with this node I get the error:
 ```
error reason=Failed to update bonds in db

Caused by:
    number of parameters must be between 0 and 65535

Error: Database
```

But if I start node and indexer from scratch everything works fine, seems this is caused by too many args when the database updated, the issue fired after this log: https://github.com/anoma/namada-indexer/blob/main/chain/src/main.rs#L432 in chain indexer

**Comment by joel-u410** [comment]
I think this was fixed by https://github.com/anoma/namada-indexer/pull/178 ?

**Comment by bombermine3** [comment]
> I think this was fixed by #178 ?

I just encountered this problem. It is not fixed:

```
chain-1  | 2024-12-11T22:11:18.589788Z  INFO chain: Inserting initial data... block=101762
chain-1  | 2024-12-11T22:11:24.467270Z ERROR shared::error: Database error reason=Failed to update bonds in db
chain-1  | 
chain-1  | Caused by:
chain-1  |     number of parameters must be between 0 and 65535
chain-1  |     
chain-1  | 2024-12-11T22:11:26.004564Z  INFO chain: Inserting initial data... block=101763
chain-1  | 2024-12-11T22:11:31.590512Z ERROR shared::error: Database error reason=Failed to update bonds in db
chain-1  | 
chain-1  | Caused by:
chain-1  |     number of parameters must be between 0 and 65535
```

**Comment by bombermine3** [comment]
There is no error. It was just necessary to do a rebuild
